Example/RxRealm_Tests/TestModels.swift

```diff
@@ -24,15 +24,7 @@ class Message: Object {
     }
 }
 
-extension Array where Element: Message {
-    func equalTo(to: [Message]) -> Bool {
-        guard count == to.count else {return false}
-        let (result, _) = reduce((true, 0)) {acc, el in
-            guard acc.0 && self[acc.1] == to[acc.1] else {return (false, 0)}
-            return (true, acc.1+1)
-        }
-        return result
-    }
+extension Message: Equatable {
 }
```

Comment:
For whatever reason Adding `Equatable` to `Message` was failing for me when I was doing the last update ... are the tests succeeding for you?